pulseapi/profiles/models.py

```diff
@@ -207,6 +201,89 @@ def thumbnail_image_tag(self):
         blank=True
     )
 
+    # --- extended information ---
+
+    enable_extended_information = models.BooleanField(
+        default=False
+    )
+
+    # currently not captured as Program class,
+    # instead captured as type+year
+
+    PROGRAM_NONE = "None"
```

Comment:
any reason we're enforcing choices at the db-level vs. at the serializer/form-level? Doing it at the latter will give us some flexibility.

Reply:
"work", mostly. Doing it at the model level is what I see as the conventional way to do this, but if you have an example of doing this at the Serializer level, I'll happily change it.

Reply:
yeah I'm not picky. It is slightly more code since you have to do it to the serializer AND the admin.

Reply:
Can you explain what you mean with that? As far as I can tell, just specifying it in the model means that it's used in building the UI without any need for modifying the Serializer

Reply:
sorry, my second comment was meant for if you do it at the serializer and admin level instead of at the model level.

Reply:
In this case I'd say leave it in the model, mostly because the "nothing selected" labels are not strings we're going to be changing a lot. Ideally, we pick ones that work for us, and then never ever touch them again. We will need to update the list of programs and years as years roll over, but that's effectively one migration a year, which feels fine.